maps/RoidStation.dmm

```diff
@@ -6958,7 +6958,7 @@
 "cDP" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/turf/simulated/wall/r_wall,/area/engineering/atmos_control)
 "cDQ" = (/obj/structure/closet/secure_closet/engineering_atmos,/obj/item/clothing/mask/gas/ert,/obj/item/weapon/tank/oxygen/yellow,/obj/effect/decal/warning_stripes{tag = "icon-warning (EAST)"; icon_state = "warning"; dir = 4},/obj/machinery/alarm{dir = 4; pixel_x = -22},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/turf/simulated/floor,/area/engineering/atmos_control)
 "cDR" = (/obj/machinery/disposal,/obj/structure/disposalpipe/trunk,/turf/simulated/floor{icon_state = "bar"},/area/hallway/primary/central)
-"cDS" = (/obj/structure/table/woodentable,/obj/item/device/illegalradio/nanotrasen,/turf/simulated/floor/carpet,/area/bridge)
```

Comment:
sneaky mapping

Reply:
>Removed selling to NPC's, removed SPS alerts, removed advanced uplinks from all bridges

Reply:
Why is there now a donk pocket voucher here?